week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -8,7 +8,9 @@
   NOTE: don't include any element that is not a "family".
 */
 
-function colonisers() {
+function colonisers(arr) {
+  let stay = arr.filter(col => col.charAt(0) === "A" && col.includes("family"));
```

Comment:
This works, but is slightly more code than we need to write to solve the problem. We're not really using the `stay` variable here, so we may not want to declare it at all, and instead just write:
`return arr.filter(col => col.charAt(0) === "A" && col.includes("family"));`
because it's a little shorter. Your code is doing the right thing, so the next level up is to think about how we can make our code as simple and easy to read as possible :)

You've done the same thing in most of the exercises, which is fine, but probably worth thinking about in the future :)